examplenuget/App.tsx

```diff
@@ -101,57 +101,10 @@ const App = () => {
             }}
           />
 
-          <Button
-            onTapped={a => {setShowState(ContentDialogState.Popup);}}
-            content="click to open a ContentDialog" />
+        <WinUI.ColorPicker onColorChanged={(e) => {
+          console.log(e.nativeEvent);
+        }} />
 
-          <ContentDialog
```

Comment:
Does this part of the example no longer work? Or just no longer matter?

Reply:
it's just a playground kind of app, so the code in here doesn't quite matter :)